src/app/controllers/admin-forms.server.controller.js

```diff
@@ -746,5 +746,33 @@ function makeModule(connection) {
         },
       )
     },
+
+    /**
+     * Transfer a form to another user
+     * @param  {Object} req - Express request object
+     * @param  {Object} res - Express response object
+     */
+    transferOwner: async function (req, res) {
+      const newOwnerEmail = req.body.email
+
+      // Transfer owner and Save the form
+      try {
+        await req.form.transferOwner(req.session.user, newOwnerEmail)
+      } catch (err) {
+        logger.error(
```

Comment:
we have changed the way the logger works. Please follow the following format:
```
logger.error({
          message: 'Failed to retrieve example forms',
          meta: {
            action: 'getExampleFormsUsingAggregateCollection',
            ip: getRequestIp(req),
            url: req.url,
            headers: req.headers,
          },
          error,
        })
```

Reply:
done